### ISSUE TITLE
Feature Request: Bookmark Friendly Vomnibar

### ISSUE THREAD

**Issue by fredfortier** [philc/vimium#3304]
As a bookmark user, I find that Vomnibar is lacking key features that lock me into using the browser's bookmark manager (bad GUI that does not play nice with Vimium). I would use Vomnibar to navigate bookmarks if it had these two key features:

1) A config option that would allow bookmarks to appear before history. Ideally, I would choose the order and the number of entries. I would also be ok with a dedicated custom search command (e.g. `b: my bookmark`).
2) Show bookmark folder path next to each bookmark entry.

Nice to have: An "add bookmark" shortcut that allows me to auto-complete the folder instead of navigating through it. Currently, adding a bookmark always requires multiple keystrokes (`Ctrl+D`, `Tab`, `Enter`, `Arrow Keys`, `Enter`, `Enter`). I'd like to get it down to (`[single key shortcut]`, `[auto-complete]`, `Enter If results == 1 else `jk+Enter`.

**Comment by gdh1995** [comment]
1. currently Vimium sorts suggestions by last visit time and text matching score, if it's not what you want, Vimium has a command `Vomnibar.activateBookmarks` and `Vomnibar.activateBookmarksInNewTab` to complete query words using only bookmark info and you may take a try.
2. it just needs a "`/`" character in your query words, and Vimium will think you need to complete using both path and title if this character is found.

**Comment by davidgenesiscruz** [comment]
I agree it would be very nice if we could navigate through the bookmark directory structure using the Vomnibar
